TCAT/AppDelegate.swift

```diff
@@ -115,6 +115,36 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
     }
     
+    @objc func storeServicedRoutes() {
+        Network.getAlerts().perform(withSuccess: { (alertRequest) in
+            if alertRequest.success {
+                if let currentRoutes = self.userDefaults.array(forKey: Constants.UserDefaults.servicedRoutes) as? [Int] {
+                    let alertRoutes = self.allRoutesInAlerts(alertRequest.data)
```

Comment:
It looks like this can be simplified to always write alertRoutes if they exist. No need to check if they're the same, you can just overwrite if they are in that case. Let me know if I'm missing something here.

Reply:
Oh this stuff was actually for keeping information about alerts globally to later present but I don't think we need to deal with that on this pr. We kinda scratched a lot of the ideas about presenting them around the app.